internal/app/secretless/handlers/mysql/handler.go

```diff
@@ -11,48 +11,127 @@ import (
 	plugin_v1 "github.com/cyberark/secretless-broker/pkg/secretless/plugin/v1"
 )
 
-// BackendConfig stores the connection info to the real backend database.
+// ConnectionDetails stores the connection info to the real backend database.
 // These values are pulled from the handler credentials config
-type BackendConfig struct {
+type ConnectionDetails struct {
 	Host     string
 	Port     uint
 	Username string
 	Password string
 	Options  map[string]string
 }
 
-func (bc *BackendConfig) Address() string {
-	return bc.Host + ":" + strconv.FormatUint(uint64(bc.Port), 10)
+// TODO: all these comments will be moved and subsumed by kumbi's work
```

Comment:
it's not clear to me here which parts are going to be taken care of by KT's changes. are we just referring to `mysqlPacket` here?

Reply:
Yes all the details about the packet structure will exist in that code.  The code you're reviewing now now is "client code" -- it will interact only with our custom Go types that represent these things, and won't know anything about the internals of packet structure.  

All this will be documented.